src/modules/logbook/LogBookTypes.ts

```diff
@@ -24,6 +24,10 @@ export const LogBookTypes: Record<string, LogBookType> = {
         display: 'primary',
         label: 'FOUND',
     },
+    ROAMER: {
+        display: 'info',
+        label: 'ROAMER',
+    },
```

Comment:
This might be too many Logbook filters. Will probably just put the roamer log under a different filter before this is finished.